File: perl/unit_test/demo/calculator_test.pl
#!/usr/bin/perl
use strict;
use warnings;

require './calculator.pl';

use Test::More tests => 2;

is(add(2, 3), 5, 'add 2 + 3');

is(subtract(5, 3), 2, 'subtract 5 - 3');

print "All tests passed.\n";

File: _experimental/_julia/unit_test/demo/test/calculator_test.jl
#!/usr/bin/julia

using Test
include("../src/Calculator.jl")
using .Calculator

@testset "Calculator Tests" begin
  @testset "Addition Tests" begin
    @test add(2, 3) == 5
  end
  
  @testset "Subtraction Tests" begin
    @test subtract(5, 2) == 3
  end
end